Broken @types/angular

Heya,

Currently, `@types/angular` has a dependency on an earlier version of itself as can be seen in https://npmcdn.com/@types/angular@1.5.5-alpha/package.json.

This cause a duplicate definition error when compiling a project that uses it:

```
kamik@T460p MINGW64 /D/sandbox/typescript-types-bug (master)
$ npm run tsc

> typescript-types-bug@1.0.0 tsc D:\sandbox\typescript-types-bug
> tsc

node_modules/@types/angular/index.d.ts(17,21): error TS2300: Duplicate identifier 'angular'.
node_modules/@types/angular/index.d.ts(18,21): error TS2300: Duplicate identifier 'ng'.
node_modules/@types/angular/node_modules/@types/angular/index.d.ts(17,21): error TS2300: Duplicate identifier 'angular'.
node_modules/@types/angular/node_modules/@types/angular/index.d.ts(18,21): error TS2300: Duplicate identifier 'ng'.

(...)
kamik@T460p MINGW64 /D/sandbox/typescript-types-bug (master)
$ npm ls
typescript-types-bug@1.0.0 D:\sandbox\typescript-types-bug
+-- @types/angular@1.5.5-alpha
| +-- UNMET DEPENDENCY @types/angular@1.5.4-alpha
| `-- @types/jquery@1.10.26-alpha
`-- typescript@2.0.0

npm ERR! missing: @types/angular@1.5.4-alpha, required by @types/angular@1.5.5-alpha
```

I've tried looking at https://github.com/DefinitelyTyped/DefinitelyTyped/tree/types-2.0/angular but there is no `package.json` there. Any way I can contribute a fix?

--- a/angular/angular-mocks.d.ts
+++ b/angular/angular-mocks.d.ts
@@ -3,7 +3,7 @@
 // Definitions by: Diego Vilar <http://github.com/diegovilar>, Tony Curtis <http://github.com/daltin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import * as angular from 'angular';
+import * as angular from './index';
 
 declare module "angular-mocks/ngMock" {
     var _: string;

--- a/angular/tsconfig.json
+++ b/angular/tsconfig.json
@@ -14,6 +14,6 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": false
     }
 }

